modules/ROOT/nav.adoc

```diff
@@ -21,9 +21,9 @@
  ** xref:import-api-specification-design-center.adoc[From Design Center]
 
 * xref:api-development-studio.adoc[API Development in Studio]
-
-* xref:create-api-specification-studio.adoc[Create an API Specification Project in Studio]
-* xref:test-specification-api-console.adoc[Test your API Specification Using the API Console]
+ ** xref:create-api-specification-studio.adoc[Create an API Specification Project in Studio]
```

Comment:
hard-coded link text necessary?

Reply:
In the Studio nav, the titles are imperative. There's a WI in my backlog waiting to fix this 🤞 
Thanks!